### PR TITLE
Mimir 672 manual refresh statreg

### PR DESCRIPTION
**Pull request by erikgus**

### CODE REVIEW

**Review by johnnadeluy — CHANGES_REQUESTED** [review]
After pressing the refresh button for statreg, is the green popup supposed to be empty?

File: src/test/resources/lib/ssb/keyFigure-test.es6 [hunk 1]
```diff
@@ -0,0 +1,54 @@
+const test = __non_webpack_require__('/lib/xp/testing')
```
[the remark this answers is not in the document]
Comment (erikgus) [reply src/test/resources/lib/ssb/keyFigure-test.es6:1]: Yes, snuck itself in. removed now.

File: src/main/resources/react4xp/_entries/Dashboard/Dashboard.jsx [hunk 2]
```diff
@@ -274,8 +277,37 @@ class Dashboard extends React.Component {
                       buttonText={`Oppdater alle tabeller (${tableQueries.length})`}
                     />
                   }
-                </Col>
-              </Row>
+                </div>
```
[the remark this answers is not in the document]
Comment (erikgus) [reply src/main/resources/react4xp/_entries/Dashboard/Dashboard.jsx:280]: removed

File: src/main/resources/react4xp/_entries/Dashboard/RefreshDataButton.jsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+      console.log('trying to reach', statregDashboardServiceUrl);
+      axios.get(statregDashboardServiceUrl)
+        .then((response) => {
+          if (onSuccess) {
+            onSuccess(response.data.message)
+          }
+        })
+        .catch((err) => {
+          console.error(err)
+          const message = err
```
[the remark this answers is not in the document]
Comment (erikgus) [reply src/main/resources/react4xp/_entries/Dashboard/RefreshDataButton.jsx]: fixd

**Review by johnnadeluy — APPROVED** [review]
Good work! 🙌

**Pull request merged**